sqs_workers/processors.py

```diff
@@ -59,11 +59,49 @@ def process_message(self, message):
         else:
             return True
 
+    def process_messages(self, messages):
+        """
+        Take a list of messages and send them in a batch
+        to the decorated call handler function.
+        """
+        message_ids = [m.message_id for m in messages]
+        extra = {
+            "message_ids": message_ids,
+            "queue_name": self.queue.name,
+            "job_name": self.job_name,
+        }
+        logger.debug(
+            "Processing batch for {queue_name}.{job_name}".format(**extra), extra=extra
+        )
```

Comment:
Why not go even simplier:

```suggestion
        logger.debug("Processing batch", extra=extra)
```

Reply:
Would clean up the logging statements for sure, I'm keen for this, happy to go back and change again.

@Schnouki thoughts?

Reply:
Sure! That's why we have `extra` :)